### PR TITLE
📄Events - .NET Superpowers

### LINKED ISSUE #596: 📄 Events - .NET Superpowers
https://www.ssw.com.au/ssw/Events/Training/NET-Superpowers-Tour.aspx

As per my conversation with @camillars and @bradystroud , we think all SuperPower pages should be under the route of /events and Internship will remain under /training and remove all the content related to COVID on the page. 
```[tasklist]
### Tasks
- [x] Migrate pages
- [x] Update MegaMenu
- [x] Add redirect (blocked by #1087)
```

### PR DESCRIPTION
**Pull request by KristenHu**
Closed #596 , merge the net superpower tour page.

Affected routes:

- `/events/net-superpowers-tour`

Add done video, screenshots
image

### CODE REVIEW

**Review by amankumarrr — CHANGES_REQUESTED** [review]
image

Dates are not same as on actual site

**Review by amankumarrr — CHANGES_REQUESTED** [review]
image

There are 5 presenters on the page

**Review by amankumarrr — CHANGES_REQUESTED** [review]
image

This content should be above the video and also some of the content is missing ther e

**Review by amankumarrr — APPROVED** [review]
LGTM 🌟

File: content/events/net-superpowers-tour.mdx [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+            icon={5}
+          />
+      - placeholder: Stage 6
+        body: |
+          <VerticalListItem
+            content={<>
+              Cloudy with a Chance of Mobile
+            </>}
+            afterBody={<>
+              * write once and run anywhere with .NET
```
Comment (Harry-Ross) [inline content/events/net-superpowers-tour.mdx]: Capitalise this word :)

**Review by Harry-Ross — APPROVED** [review]
Other than the small typo, LGTM

File: content/events/net-superpowers-tour.mdx (hunk 1 above)
Comment (KristenHu) [reply content/events/net-superpowers-tour.mdx]: Thanks, fixed😃

**Pull request merged**